### DISCUSSION TITLE
Persistent Outlook icon and other

### DISCUSSION THREAD

**Discussion by tsyrstad73** [kangyu-california/PersistentWindows#304] [Q&A]
Recently I noticed that I have a Persistent Outlook and Persistent Chrome icon in my taskbar icons. When I turn them off, Persistent Windows also turns off. Turn Windows back on and Outlook and Chrome are also back. These are not the true Outlook and Chrome task icons - those are completely separate. These appear to be tied to the PW program. I like to have the PW icon so I can see when it's red (do not touch pc!). Why do I have these other 2 and can I get rid of them? They are confusing. -thanks
![image](https://github.com/kangyu-california/PersistentWindows/assets/115507119/b8b38b47-b915-48cc-9484-e0d378a15bde)

**Comment by kangyu-california** [comment]
Those icons are indeed created by PersistentWindows from "Google Chrome" and "Outlook" app icon when user Ctrl clicked minimize button, which is interpreted by PW to hide the original app icon from taskbar and substitute the "faked" one into Systray.

The implementation seems like a hack, because Microsoft does not provide an API to simply move the icon from taskbar to systray.

I believe it is a safety measure at OS level to disallow faking the creator of those icons.

**Reply by tsyrstad73** [reply]
I’ve never Ctrl clicked to minimize a button. And the icons are still in my taskbar.

**Reply by kangyu-california** [reply]
Try disable the feature by adding following command argument when launching PW,
 -ctrl_minimize_to_tray=0

**Reply by tsyrstad73** [reply]
ok, I did that, but then the PW icon was also missing. I'm not sure if perhaps I was doing multiple arguments incorrectly. I couldn't find an example on the help page. I ended up rebooting - which went really slow and I had a black screen. My desktop failed to generate properly so I did another restart which went normal. I then put an argument value of 1, thinking I had turned off all PW icons with the former 0. I started PW and saw there was a new update and installed that. And now everything is back to normal (I also put my bat file back to normal). So.. everything looks good and I have no idea what's been happening.
